clap_generate/src/man.rs

```diff
@@ -0,0 +1,287 @@
+use std::fmt::write;
```

Comment:
It'd be really nice if it was granular enough that it could be used multiple places but not sure how feasible it is given formatting and different requirements.